components/ILIAS/Mail/classes/class.ilObjMailGUI.php

```diff
@@ -511,6 +517,16 @@ protected function getExternalSettingsForm(): ilPropertyFormGUI
         $user_envelope_from_addr->setDisabled(!$this->isEditingAllowed());
         $form->addItem($user_envelope_from_addr);
 
+        $signature = new ilTextAreaInputGUI(
+            $this->lng->txt('mail_system_sys_general_signature'),
+            'mail_system_usr_general_signature'
+        );
+        $signature->setRows(8);
+        $signature->setDisabled(!$this->isEditingAllowed());
+        $form->addItem($signature);
+
+        $form->addItem($this->buildSignaturePlaceholderInput((new MailUserSignature(0))));
```

Comment:
Can we use `null` here instead?

```suggestion
        $form->addItem($this->buildSignaturePlaceholderInput(new MailUserSignature(0)));
```